src/content/glossary/index.md

```diff
@@ -470,6 +567,10 @@ Also known as a "password stretching algorithm," it is used by [keystore](#keyst
   Smart contract security
 </DocLink>
 
+### keyfile {#keyfile}
+
+Every account’s private key/address pair exists as a single keyfile in an Ethereum client. These are JSON text files which contains the encrypted private key of the account, which can only be decrypted with the password entered during account creation.
+
```

Comment:
Is the term "keyfile" used very often? Feel like "keystore" is much more common. 

Also, this is confusing and makes it sound like all the private/public keys to all accounts exist inside the client software. 

Feel like rewording to not mention Ethereum "clients" here, but perhaps "wallet" or "wallet client" instead?

Reply:
keyfiles are kept in a  keystore...

but yea maybe keystore is the better term to include instead of keyfile. Will update.